src/components/donation/Steps/DonateMethods/Stripe/Stripe.tsx

```diff
@@ -0,0 +1,125 @@
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+import { useState } from "react";
+import { FormValues, Props } from "./types";
+import { useCreateStripePaymentIntentMutation } from "services/apes";
+import { useErrorContext } from "contexts/ErrorContext";
+import KYCForm from "components/KYCForm";
+import { KYC } from "slices/donation";
+import { PUBLIC_STRIPE_KEY } from "constants/env";
+import Checkout from "./Checkout";
+import Form from "./Form";
+
+const stripePromise = loadStripe(PUBLIC_STRIPE_KEY);
+
+// Followed Stripe's custom flow docs
+// https://stripe.com/docs/payments/quickstart
+export default function Stripe(props: Props) {
+  const [step, setStep] = useState<Step>({ type: "init" });
+
+  const { handleError } = useErrorContext();
+  const [createPaymentIntent] = useCreateStripePaymentIntentMutation();
+
+  switch (step.type) {
+    case "init":
+      const { type, kycData, ...defaultValues } = step;
+      return (
+        <Form
+          {...props}
+          defaultValues={defaultValues}
+          onSubmit={async (fv) => {
+            if (fv.userOptForKYC) {
+              return setStep({ ...step, type: "kyc", ...fv });
+            }
+            try {
+              const { clientSecret } = await createPaymentIntent({
+                amount: fv.amount,
+                currency: fv.currency.code,
+                endowmentId: props.state.recipient.id,
+                splitLiq: fv.pctLiquidSplit.toString(),
+              }).unwrap();
+              setStep({ ...step, type: "checkout", clientSecret, ...fv });
+            } catch (err) {
+              handleError(err, "Failed to load payment platform");
+            }
+          }}
+        />
+      );
+    case "kyc":
+      return (
+        <KYCForm
```

Comment:
kyc looks cramped

<img width="524" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/b9814b43-b981-4a7a-8d1e-80ba52b7b7cb">

Reply:
It does look ugly.
Copied classes used in donation `CurrentStep.tsx`, looks much better.
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/87d7c7d4-be41-482a-888e-4b815ed23bae)